File: TypeScript/palindrome_string.ts
export function is_palindrome_string(word: string) {
  const MINUS_ONE = 1;
  let is_palindrome = true;

  for (let index = 0; index < word.length / 2; index++) {
    const current_char = word[index].toLowerCase();
    const opposite_char = word[word.length - index - MINUS_ONE].toLowerCase();

    if (current_char !== opposite_char) {
      is_palindrome = false;
      break;
    }
  }

  if (is_palindrome) {
    console.log("La palabara %o es un palindromo", word);
  } else {
    console.log("La palabara %o no es un palindromo", word);
  }

  return is_palindrome;
}

File: Python/palindrome_string.py
def is_palindrome_string(word: str):
    MINUS_ONE = 1
    is_palindrome = True

    for index in range(round(len(word) / 2)):
        current_word = word[index].lower()
        opposite_char = word[len(word) - index - MINUS_ONE].lower()

        if current_word != opposite_char:
            is_palindrome = False
            break

    if is_palindrome:
        print("La palabara {} es un palindromo".format(word))
    else:
        print("La palabara {} no es un palindromo".format(word))

    return is_palindrome
